tequilapi/endpoints/service.go

```diff
@@ -56,28 +52,32 @@ type serviceList []serviceInfo
 // swagger:model ServiceInfoDTO
 type serviceInfo struct {
 	// example: 6ba7b810-9dad-11d1-80b4-00c04fd430c8
-	ID       string      `json:"id"`
+	ID string `json:"id"`
+
 	Proposal proposalRes `json:"proposal"`
-	// example: Running
-	Status  string         `json:"status"`
-	Options serviceOptions `json:"options"`
```

Comment:
If we are removing "Options" from the service info, what should be the way to figure out what options were used for starting a service?

Reply:
Anyways it was not implemented, I am just removing dead code here

Reply:
Registered to ticket #764